commands/host.vegman

```diff
@@ -158,25 +158,66 @@ function cmd_virtualmedia {
   subcommand "$@"
 }
 
+# @sudo cmd_virtualmedia_status admin
+# @restrict cmd_virtualmedia_status admin
+# @doc cmd_virtualmedia_status
+# Get status of the virtual media
+function cmd_virtualmedia_status {
+  local usb_status_metadata=( "USB0;Proxy" "USB1;Legacy" )
+
+  for usb_item_metadata in "${usb_status_metadata[@]}"
+  do
+    local id="${usb_item_metadata%%;*}"
+    local vm_type="${usb_item_metadata#*;}"
+    local usb_status=$(busctl get-property xyz.openbmc_project.VirtualMedia \
+                                        /xyz/openbmc_project/VirtualMedia/${vm_type}/${id} \
+                                        xyz.openbmc_project.VirtualMedia.Process Active)
+    local image=""
+    echo -ne "Virtual Media ($id) status:\t"
+    if [ "$usb_status" == "b true" ]; then
+      echo "mounted"
+      image=$(busctl get-property  xyz.openbmc_project.VirtualMedia \
+                                  /xyz/openbmc_project/VirtualMedia/${vm_type}/${id} \
+                                  xyz.openbmc_project.VirtualMedia.MountPoint ImageURL)
+      image=${image#*\"}
+      echo -e "\tImage: ${image%\"*}"
```

Comment:
This outputs:
```
Virtual Media (USB0) status: mounted
        Image: protocol://some.domain.tld/path/to/image.bin
```

I don't see why this `\t` is there. It just consumes space. I guess this reads better:
```
Virtual Media (USB0) status: mounted
Image: protocol://some.domain.tld/path/to/image.bin
```
Please remove `\t`.

Reply:
Done, now the output is
```
vegman-s320-2007210001:~$ host virtualmedia status           
Virtual Media (USB0) status: unmounted
Virtual Media (USB1) status: mounted
Image: https://ftp.uni-erlangen.de/mirrors/ubuntu-releases/20.04/ubuntu-20.04.2.0-desktop-amd64.iso

```